server/s3_object_metadata.cc

```diff
@@ -35,29 +37,31 @@
 
 extern struct m0_uint128 global_instance_id;
 
-void S3ObjectMetadata::initialize(bool ismultipart, std::string uploadid) {
-  json_parsing_error = false;
+S3ObjectMetadata::S3ObjectMetadata(const S3ObjectMetadata& src)
```

Comment:
I guess you have written this copy constructor to copy source object to destination object. But please make sure the owner and other things are set to right values, and not same as source object.

Reply:
Am working on this just now